x-pack/plugins/security/public/management/api_keys/api_keys_grid/empty_prompt/empty_prompt.tsx

```diff
@@ -18,7 +18,7 @@ export const EmptyPrompt: React.FunctionComponent<Props> = ({ isAdmin, docLinks
   <EuiEmptyPrompt
     iconType="managementApp"
     title={
-      <h1>
+      <h1 data-test-subj="NoApiKeysHeader">
```

Comment:
nit: data test subjects should be `camelCase` ([style guide](https://github.com/elastic/kibana/blob/e3535b1b1c7f66251a2d422a218615449fe8190d/STYLEGUIDE.md#L43-L51)):
```suggestion
      <h1 data-test-subj="noApiKeysHeader">
```